src/structure/pdb.jl

```diff
@@ -107,6 +321,17 @@ function Base.read(filepath::AbstractString,
     end
 end
 
+# Read PDB file based on PDB ID and pdb_dir. 
+function Base.read(pdbid::AbstractString,
```

Comment:
This method clashes with the above `read` method. I wonder whether if it is necessary, but if it goes in the directory should be a full argument.